### PR TITLE
MNT: Add noarch: python

### PR DESCRIPTION
**Pull request by conda-forge-linter**
Hi! This is the friendly automated conda-forge-webservice.

I've made the recipe `noarch: python` as instructed in #21.

Here's a checklist to do before merging.
- [ ] Bump the build number if needed.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -9,6 +9,7 @@ source:
   sha256: 74eb96b4e1eb08fbe37b5c6fbad9c978e1c9461013378f5df7637c2d8eccb18e
 
 build:
+  noarch: python
   number: 0
```
Comment (nsoranzo) [inline recipe/meta.yaml:13]:
```suggestion
  number: 1
```

**Comment by ocefpaf** [comment]
I don't think this can be `noarch`. I have a vague memory of trying and had to revert it.

**Comment by nsoranzo** [comment]
> I don't think this can be `noarch`. I have a vague memory of trying and had to revert it.

There's no trace of `noarch` in the git log or in any PR. Is there a way to test the `noarch` packages built on Azure?

**Comment by ocefpaf** [comment]
> There's no trace of `noarch` in the git log or in any PR. Is there a way to test the `noarch` packages built on Azure?

Nope. I remember building it on linux and trying to install on Windows. The situation may have changed though. Just go for it and we can pull the packages if things are broken.

**Comment by nsoranzo** [comment]
> Just go for it and we can pull the packages if things are broken.

I am not a maintainer of this feedstock, so I cannot merge, if that's what you meant. I may add myself in #23 though.
By the way, I can cherry pick these commits into https://github.com/conda-forge/virtualenv-feedstock/pull/23 to prevent merge conflicts, if you want.

**Comment by mingwandroid** [comment]
Out of interest, what was the end result of this? I cannot really think of a reason virtualenv couldn't be noarch.

**Comment by nsoranzo** [comment]
@mingwandroid I tested under Linux, which predictably worked, while @tonietuk kindly tested it under OSX, also successfully.

**Pull request merged**